recipe/build.sh

```diff
@@ -43,11 +43,16 @@ cmake ${CMAKE_ARGS} -GNinja \
       -DENABLE_CCACHE=OFF \
       -DBUILD_WITH_OPENMP=OFF \
       -DFORCE_RSUSB_BACKEND=ON \
+      -DBUILD_PYTHON_BINDINGS:bool=true \
+      -DPYTHON_EXECUTABLE="$PREFIX/bin/python" \
       -DBUILD_EXAMPLES=OFF \
       -DBUILD_UNIT_TESTS=OFF \
       -DCHECK_FOR_UPDATES=OFF \
       -DCMAKE_VERBOSE_MAKEFILE=ON \
       $SRC_DIR
 
 cmake --build . --config Release 
-cmake --install . --config Release 
+cmake --install . --config Release
+
+# Copy pyrealsense files to site-packages (there is probably a bettter way to do this with -DPYTHON_INSTALL_DIR)
+mv $PREFIX/OFF/* $PREFIX/lib/python3.[0-9]/site-packages/
```

Comment:
Just to understand, how comes that the python files end in the `$PREFIX/OFF/` directory? Also, for reference in this build scripts you can also use directly the `$SP_DIR`  env variable, see https://docs.conda.io/projects/conda-build/en/stable/user-guide/environment-variables.html#environment-variables-set-during-the-build-process .

Reply:
I'm not sure why it ends up in `$PREFIX/OFF/`, my assumption is it's because of a CMake variable which defaults to `OFF` but I couldn't quite figure out which that would be since we already specify all the Python ones it seems. Thanks for the `$SP_DIR` hint, that was what I was looking for

Reply:
Turns out it likely was PYTHON_INSTALL_DIR, resolved in latest commit